corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -138,12 +129,22 @@ def input_trans(name, langs=None, input_name='name', input_id=None, data_bind=No
                    placeholder="{placeholder}" />
         '''
     elif element_type == 'textarea':
+        options = _get_dynamic_input_trans_options(name, langs=langs, value_escape="html")
```

Comment:
Why does a `<textarea>` want different escaping than an `<input>` textbox?

Reply:
{value} inside a text area was being js escaped and that string is directly used in the html template. So I was having an issue where "M&M" would be displayed as "M//u0026M". The html escape escapes the & such that it displays correctly.